docs/program/extend/custom-components-remotes.md

```diff
@@ -4,19 +4,21 @@ linkTitle: "Custom Components as Remotes"
 weight: 99
 type: "docs"
 tags: ["server", "sdk"]
+aliases:
+    - "program/extend/custom-components-remotes"
```

Comment:
```suggestion
    - "program/extend/sdk-as-server"
```
The alias must be the old page link, that's what will then be forwarded to this one

Reply:
done